src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,23 @@
+package com.example;
+
+/**
+ * Public class Customer with the field loyaltyCard and
+ * constructor
+ *
+ * @author Elena Yevsyukova
+ * @version 2.1
+ */
+public final class Customer {
+    /**
+     *The private field of type String
```

Comment:
Of course, it is. I can see it from the code! Why are we having this field? What does it mean from the domain perspective?